includes/admin/class-wp-job-manager-settings.php

```diff
@@ -829,15 +828,43 @@ protected function input_radio( $option, $ignored_attributes, $value, $ignored_p
 		if ( ! empty( $option['desc'] ) ) {
 			echo ' <p class="description">' . wp_kses_post( $option['desc'] ) . '</p>';
 		}
-
-		foreach ( $option['options'] as $key => $name ) {
-			echo '<label><input name="' . esc_attr( $option['name'] ) . '" type="radio" value="' . esc_attr( $key ) . '" ' . checked( $value, $key, false ) . ' />' . esc_html( $name ) . '</label><br>';
+		if ( 'checkbox' === $type ) {
+			foreach ( $option['options'] as $key => $name ) {
+				$value = (array) $value;
+				echo '<label><input name="' . esc_attr( $option['name'] . '[fields][]' ) . '" type="' . esc_attr( $type ) . '" value="' . esc_attr( strtolower( $name ) ) . '" ' . checked( isset( $value['fields'] ) && is_array( $value['fields'] ) && in_array( strtolower( $name ), $value['fields'], true ), true, 0 ) . '/>' . esc_html( $name ) . '</label><br>';
```

Comment:
Might be worth extracting the code that outputs the input+label into it's own function

Reply:
Thanks Panos!  What do you think about 373c5173441463a71b804b7917833ffa6381629c ?

Reply:
I like it! I would additionally probably do the checkbox check once and save it into a var but not a show stopper!